src/elm/Community.elm

```diff
@@ -142,31 +141,20 @@ communitiesSelectionSet =
         |> with Community.memberCount
 
 
-dashboardSelectionSet : SelectionSet DashboardInfo Cambiatus.Object.Community
-dashboardSelectionSet =
-    SelectionSet.succeed DashboardInfo
-        |> with Community.name
-        |> with Community.logo
-        |> with Community.memberCount
-        |> with Community.transferCount
-        |> with Community.actionCount
-        |> with Community.productCount
-        |> with Community.hasObjectives
-        |> with (Eos.nameSelectionSet Community.creator)
-        |> with (Community.validators (Eos.nameSelectionSet Profile.account))
-
-
 communitySelectionSet : SelectionSet Model Cambiatus.Object.Community
 communitySelectionSet =
     SelectionSet.succeed Model
         |> with Community.name
         |> with Community.description
         |> with (Eos.symbolSelectionSet Community.symbol)
         |> with Community.logo
+        |> with (Community.subdomain Subdomain.name |> SelectionSet.map (Maybe.withDefault ""))
```

Comment:
oh, this is a maybe? not good, subdomain.name can't be null. I'll fix the type

Reply:
I thought you said we needed to deploy the update first :thinking:

Reply:
we do, but they are different!

the one i've mentioned before is to make all communities have an obligatory subdomínio, but here is about the name not being null

Reply:
or am i reading this code wrong? 🤔 maybe so..

this is like this because the name can be null right?

Reply:
Oh, right, makes sense

Reply:
Actually, subdomain can be null, but subdomain.name is always a string:

![2021-05-18-083531_461x85_scrot](https://user-images.githubusercontent.com/39280468/118644264-058c2900-b7b4-11eb-9f6a-b934b677e015.png)

Reply:
oh ok, i've misread the code then